### ISSUE TITLE
Complain about footway/railway intersection without provided info about legality of crossing there

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#9310]
### Description

To replicate: 

map/find `railway=rail`
map/find `highway=footway` or `highway=path` and do not mark it as `access=no` - attach it to an existing road then create shared node with railway

Expected:

iD asks mapper to provide info is it illegal crossing (`crossing=no` on shared node, ideally `access=no` on footway section) or is it signed legal crossing (`railway=crossing`)

Can be skipped on ways with `access=no`

Why it is useful?

In some cases OSM maps existing illegal crossings without marking them as illegal. While I occasionally crossed railway tracks and it may make sense in some cases, such thing definitely should be marked.

And typical router engine would prefer to avoid routing over illegal railway crossings and providing such info is a good idea.

https://wiki.openstreetmap.org/wiki/Tag:crossing%3Dno (was also mentioned at https://github.com/openstreetmap/id-tagging-schema/issues/548 - note that "most (90%) of the usages of this tag are in combination with highway=traffic_signals")

See https://overpass-turbo.eu/s/1mhm for such problematic railway-path intersections.

See https://overpass-turbo.eu/s/1mhn for all such problematic intersections, also rail-road ones.

See say https://www.radiolodz.pl/posts/27234-zamkneli-przejscie-przez-tory-zgierzanie-protestuja for a clear unambiguous illegal path that can be mapped but should be marked as illegal

![13898617_1249440445080709_1063680557_o](https://user-images.githubusercontent.com/899988/180623990-3e0443e6-adaa-4879-97ec-6810022f44c3.jpg)

### Screenshots

![screen08](https://user-images.githubusercontent.com/899988/192884297-15b1f163-edc8-4417-90b4-d31f2790d5e5.png)

**Comment by matkoniecz** [comment]
See also #9306 for why I think that trams should be excluded. But some `railway=rail` is street running what may cause the same issues...

**Comment by tyrasd** [comment]
Agree, there should be a second option to mark a crossing as "illegal". Probably ideally mapped as a short path segment with `access=no` and optionally also `crossing=no` on the vertex.
